docs/zack-jones.md

```diff
@@ -0,0 +1,2 @@
+# Zack-Jones Bio
+- Name: Zack Jones
```

Comment:
There should be more information to add here that is found in the corresponding issue.

More coming soon!